test/run/ch-run_sqfs.bats

```diff
@@ -0,0 +1,21 @@
+load ../common
+
+@test 'ch-run: squash' {
```

Comment:
I think we probably do not need a separate Bats file but rather should send the whole test suite through your code. We can talk about how to make that happen; I've thought about it a bit but would need to refresh my memory.